ports/coreMQTT/azure_iot_core_mqtt.c

```diff
@@ -0,0 +1,233 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved. */
+/* SPDX-License-Identifier: MIT */
+
+/**
+ * @file azure_iot_core_mqtt.c
+ * @brief Implements the port for Azure IoT MQTT based on coreMQTT.
+ * 
+ */
+ 
+#include <assert.h>
+
+#include "azure_iot_mqtt.h"
+
+/**
+ * Maps CoreMQTT errors to AzureIoTMQTT errors.
+ **/
+static AzureIoTMQTTResult_t prvGetAzureIoTMQTTResult( MQTTStatus_t xResult )
```

Comment:
Nit (because who doesn't love all these nits :))? - I think instead of Get this should be Translate, because Get usually either implies a direct accessor or some sort of API call or something to retrieve data.